godot-core/src/builtin/aabb.rs

```diff
@@ -43,32 +43,346 @@ impl Aabb {
+        self.size.min_axis_index()
+    }
+
+    /// Returns the scalar length of the shortest axis of the AABB.
+    ///
+    /// _Godot equivalent: `AABB.get_shortest_axis_size`_
+    pub fn shortest_axis_size(&self) -> real {
+        self.size.x.min(self.size.y.min(self.size.z))
+    }
+
+    /// Returns the support point in a given direction. This is useful for collision detection algorithms.
+    ///
+    /// _Godot equivalent: `AABB.get_support()`_
+    pub fn support(&self, dir: Vector3) -> Vector3 {
+        let half_extents = self.size * 0.5;
+        let ofs = self.position + half_extents;
+
+        let x = if dir.x > 0.0 {
+            half_extents.x
+        } else {
+            -half_extents.x
+        };
+
+        let y = if dir.y > 0.0 {
+            half_extents.y
+        } else {
+            -half_extents.y
+        };
+
+        let z = if dir.z > 0.0 {
+            half_extents.z
+        } else {
+            -half_extents.z
+        };
+
+        Vector3::new(x, y, z) + ofs
+    }
+
+    /// Returns `true` if the AABB overlaps with `b` (i.e. they have at least one point in common).
+    ///
+    /// If `include_borders` is `true`, they will also be considered overlapping if their borders touch, even without intersection.
+    ///
+    /// _Godot equivalent: `AABB.intersects(AABB b, bool include_borders)`_
+    #[inline]
+    pub fn intersects(&self, &b: &Aabb, include_borders: bool) -> bool {
+        let end = self.end();
+        let end_b = b.end();
+
+        if include_borders {
+            self.position.x <= end_b.x
+                && end.x >= b.position.x
+                && self.position.y <= end_b.y
+                && end.y >= b.position.y
+                && self.position.z <= end_b.z
+        } else {
+            self.position.x < end_b.x
+                && end.x > b.position.x
+                && self.position.y < end_b.y
+                && end.y > b.position.y
+                && self.position.z < end_b.z
+                && end.z > b.position.z
+        }
+    }
+
+    /// Returns `true` if the AABB is on both sides of a plane.
+    ///
+    /// _Godot equivalent: `AABB.intersects_plane(Plane plane)`_
+    pub fn intersects_plane(&self, plane: Plane) -> bool {
+        let points = [
+            self.position,
+            self.position + Vector3::new(0.0, 0.0, self.size.z),
+            self.position + Vector3::new(0.0, self.size.y, 0.0),
+            self.position + Vector3::new(self.size.x, 0.0, 0.0),
+            self.position + Vector3::new(self.size.x, self.size.y, 0.0),
+            self.position + Vector3::new(self.size.x, 0.0, self.size.z),
+            self.position + Vector3::new(0.0, self.size.y, self.size.z),
+            self.position + self.size,
+        ];
+
+        let mut over = false;
+        let mut under = false;
+
+        for point in points {
+            // TODO: Replace when Plane.distance_to() is implemented
+            println!("{}", point);
```

Comment:
Please remove this print statement (and potential others).

Also, `Plane::distance_to()` is now implemented.

Reply:
Well that's embarrassing 😅